server/lib/pbench/__init__.py

```diff
@@ -198,6 +198,7 @@ def __init__(self, cfg_name):
             if not os.path.isdir(self.LIBDIR): raise BadConfig("Bad LIBDIR={}".format(self.LIBDIR))
 
             self.ARCHIVE = self.conf.get("pbench-server", "pbench-archive-dir")
+            self.BACKUP = self.conf.get("pbench-server", "pbench-backup-dir")
```

Comment:
Doesn't this make BACKUP a required configuration parameter?

We don't require it for all our scripts today, do we?

If the above are true, why not make this operate like `PBENCH_ENV` below?

Reply:
this needs to go. No need for back here.